db/schema.rb

```diff
@@ -196,7 +196,7 @@
   create_table "playlists", force: :cascade do |t|
     t.string "title"
     t.bigint "user_id", null: false
-    t.text "comment"
+    t.string "comment"
```

Comment:
This is a bit odd but we don't commit this change even though it was automatically made by rails.  Text vs. string can be different with different DBs (mysql vs. postgresql) and I've always been afraid that switching might cause the field to get truncated.  Please undo this change and I'll look into if it is safe to make or not.
```suggestion
    t.text "comment"
```